src/atomate2/lobster/schemas.py

```diff
@@ -0,0 +1,585 @@
+"""Module defining amset document schemas."""
+
+import logging
+import os
+import time
+from pathlib import Path
+from typing import Any, List, Optional, Union
+
+import numpy as np
+from monty.dev import requires
+from monty.serialization import loadfn
+from pydantic import BaseModel, Field
+from pymatgen.core import Structure
+from pymatgen.electronic_structure.cohp import CompleteCohp
+from pymatgen.electronic_structure.core import Spin
+from pymatgen.electronic_structure.dos import LobsterCompleteDos
+from pymatgen.io.lobster import (
+    Charge,
+    Doscar,
+    Icohplist,
+    Lobsterin,
+    Lobsterout,
+    MadelungEnergies,
+)
+
+from atomate2 import __version__
+
+# from atomate2.common.schemas.structure import StructureMetadata
+from atomate2.utils.datetime import datetime_str
+
+# from atomate2.common.files import get_zfile
+# from atomate2.utils.path import get_uri
+# from atomate2.common.schemas.math import Matrix3D, Vector3D
+
+
+try:
+    from lobsterpy.cohp.analyze import Analysis
+    from lobsterpy.cohp.describe import Description
+except ImportError:
+    Analysis = None
+    Description = None
+
+__all__ = ["LobsterTaskDocument"]
+
+logger = logging.getLogger(__name__)
+
+
+class LobsteroutModel(BaseModel):
+    """Collection to store computational settings from the LOBSTER computation."""
+
+    restart_from_projection: bool = Field(
+        "Bool indicating if the run has been restarted from a projection"
```

Comment:
Are you not missing the default value (`None`) and `description=` in this model? IIRC, the None is needed so that you can make the model without all the fields present. This is useful if you query the outputs database for only a few of the keys in the model, as otherwise there will be an error when you try and instantiate the model without all the keys present.

Reply:
I think the default value should actually be None for everything here. That allows pydantic to reconstruct the model even with fields missing. E.g. if you only query part of the model from the database

Reply:
Okay, good! I will add this! Thank you.